### DISCUSSION TITLE
B Y D

### DISCUSSION THREAD

**Discussion by zzr-shift** [Meloong-Git/PCL#4637] [讨论]
nnd 有时候非到极致也是欧
玩千万别点 有那多种可能 我点了四次 出来四次《Never Gonna Give You UP》

**Comment by SALTWOOD** [comment]
？谁不是呢，没必要为了这点事发一个讨论

**Reply by kong-que-dnf123** [reply]
~讨论不就是为了干这个的吗（他没发成ISSUE就不错了.jpg~

**Reply by E0x72-21** [reply]
> ~讨论不就是为了干这个的吗（他没发成ISSUE就不错了.jpg~

发issue没绷住）
